src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -122,6 +122,30 @@ public static String getDefaultAlias(String sourceName) {
         return sourceNameLower.substring(startIndex, endIndex);
     }
 
+    /**
+     * Returns the alias of simple check name for a check, The alias is
+     * for the form of CheckNameCheck or CheckName.
+     *
+     * @param sourceName the source name of the check (generally the class
+     *        name)
+     * @return the alias of the simple check name for the given check
+     */
+    private static String getSimpleNameAlias(String sourceName) {
+        final int checkNameStartIndex = sourceName.lastIndexOf('.') + 1;
+        final String checkName = sourceName.substring(checkNameStartIndex);
+        final String checkNameSuffix = "Check";
```

Comment:
This code supposed to exist in some utilities of checkstyle, please try to look around, if not found lete know , I will go deeper